pypom/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .page import Page  # noqa
 from .region import Region  # noqa
+from .view import WebView  # noqa
```

Comment:
Omitting this was intentional, I don't expect users to import WebView, and if they really want to, they don't need this helper.